tfhe-zk-pok/src/proofs/pke.rs

```diff
@@ -935,14 +971,13 @@ pub fn verify<G: Curve>(
     public: (&PublicParams<G>, &PublicCommit<G>),
     metadata: &[u8],
 ) -> Result<(), ()> {
-    let &Proof {
+    let Proof {
         c_hat,
         c_y,
         pi,
-        c_hat_t,
-        c_h,
-        pi_kzg,
-    } = proof;
+        compute_load_proof_fields,
+    } = proof.clone();
```

Comment:
need to clone ?

Reply:
Before this PR the code was silently cloned because all fields implement copy. The `ComputeLoadProof` does not implement copy. I need to "own" the values because a lot of the functions used later take their arguments by value.

I can derive Copy for CoputeLoadProof if you prefer.